src/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -289,6 +289,65 @@ contract BLSRegistryCoordinatorWithIndices is EIP712, Initializable, IBLSRegistr
         }
     }
 
+    /**
+     * @notice Updates the stakes of all operators for each of the specified quorums in the StakeRegistry. Each quorum also
+     * has their StakeRegistry.quorumTimestamp updated. which is meant to keep track of when operators were last all updated at once. 
+     * @param operatorsPerQuorum is an array of arrays of operators to update for each quorum. Note that each nested array
+     * of operators must be sorted in ascending operatorId order to ensure that all operators in the quorum are updated
+     * @param quorumNumbers is an array of quorum numbers to update
+     * @dev This method is used to update the stakes of all operators in a quorum at once, rather than individually
+     * 
+     */
+    function updateOperatorsForQuorum(address[][] calldata operatorsPerQuorum, bytes calldata quorumNumbers) external onlyWhenNotPaused(PAUSED_UPDATE_OPERATOR) {
+        require(
+            operatorsPerQuorum.length == quorumNumbers.length,
+            "BLSRegistryCoordinatorWithIndices.updateOperatorsForQuorum: input length mismatch"
+        );
+        for (uint256 i = 0; i < quorumNumbers.length; ++i) {
+            uint8 quorumNumber = uint8(quorumNumbers[i]);
```

Comment:
it doesn't seem strictly necessary, but we might want some kind of sanitization here on the `quorumNumbers` input?
the main thing that strikes me is that you could input an invalid quorum number? but I guess it'd effectively be a no-op

Reply:
Oh yeah, the other methods all have this check. _quorumExists

Reply:
https://github.com/Layr-Labs/eigenlayer-middleware/pull/62/commits/0556f9276126cebd4eecde019282670365a175db
Added a modifier for this now